docs/api/ethereum.md

```diff
@@ -366,6 +372,10 @@ A class instance `variable` holding a `string` which represents the user's curre
 
 A class instance `variable` holding a `string` which represents the URL to the compiled ERC-721 related smart contract definition file. This file is used when deploying new asset ledgers to the network.
 
+### deployGatewayId
+
+A class instance `variable` holding a `string` which represents an Ethereum address of the [deploy gateway](/#public-addresses).
```

Comment:
A class instance `variable` holding a `string` that represents an Ethereum address of the [deploy gateway](/#public-addresses).